techniques/general/g20x.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
+  <head>
+    <title>Adjusting indentation </title>
+    <link rel="stylesheet" href="../../css/sources.css" class="remove"/>
+  </head>
+  <body>
+    <h1>Providing options within the content to switch to a layout that does not require the user to scroll horizontally to read a line of text</h1>
```

Comment:
Do we need "horizontal" in the heading? Also, seems a little long for the technique title.

Reply:
as mentioned during the call, this is the heading from an existing technique that i had copied as the template to start working on this new one.

https://www.w3.org/WAI/WCAG22/Techniques/general/G206

This heading, along with the title, need to be updated to reflect the purpose of this new technique.